src/main/java/mandel/projectile/ProjectileGraph.java

```diff
@@ -0,0 +1,37 @@
+package mandel.projectile;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class ProjectileGraph extends JComponent
+{
+    private Projectile projectile = new Projectile(0, 0);
+
+    @Override
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+
+        g.translate(0, getHeight());
+        g.setColor(Color.BLUE);
+        g.fillOval((int) (projectile.getInterceptX() / 2) - 5, (int) (-projectile.getPeakY()) - 5, 10, 10);
+
+        int x;
+        int y;
+
+        g.setColor(Color.BLACK);
+
```

Comment:
projectile.setSeconds(0);

Otherwise the for loop can have strange results.

Reply:
Oh ok thank you, I just fixed it. Should I upload another screenshot?

Reply:
Yes.